### PR TITLE
[WIP]PED-6440: note modification and steps to enable fips

### PR DESCRIPTION
**Pull request by Amrita42**
### PR creator: Description

To modify the imp notes and add steps to enable FIPs 

### PR creator: Are there any relevant issues/feature requests?

[PED-6440](https://jira.suse.com/browse/PED-6440)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP5/openSUSE Leap 15.5
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by Amrita42** [comment]
@denknorr @pmgdeb can you please review the important note and steps to enable FIPs.

Note: This is for SP6 ,separate branches for sp5/sp4 will be created with steps to enable FIPs (the important notes are already merged )

**Comment by Amrita42** [comment]
[book-security_en.pdf](https://github.com/SUSE/doc-sle/files/14292255/book-security_en.pdf)

**Comment by Amrita42** [comment]
SP5: https://github.com/SUSE/doc-sle/pull/1660

**Comment by denknorr** [comment]
It looks good to me!

**Comment by Amrita42** [comment]
@dariavladykina please provide a peer review. Thanks

File: xml/security_fips.xml [hunk 1]
```diff
@@ -209,6 +206,50 @@
...
   </sect1>
+  <sect1 os="sles" xml:id="sec-how-to-fips-install">
+    <title>Enabling FIPS mode</title>
+   <para>
+    The following procedure shows you how to enable FIPs mode:
+    </para>
+   <procedure>
+    <step>
+<para>The crypto-policies-scripts package that provides the <literal>fips-mode-setup</literal> command might not be installed by default. If not, you can install
+the package in a supported way and then enable the kernel FIPS mode which also sets the system policy to FIPS:
```
Comment (dariavladykina) [inline xml/security_fips.xml:217]:
```suggestion
<para>The <package>crypto-policies-scripts</package> package that provides the 
<literal>fips-mode-setup</literal> command might not be installed by default. If not, 
you can install the package in a supported way and then enable the kernel FIPS mode,
which also sets the system policy to FIPS:
```

**Review by dariavladykina — COMMENTED** [review]
Hi, just a couple minor nits here. Thanks!

**Pull request merged**